pootle/apps/pootle_store/forms.py

```diff
@@ -477,9 +480,9 @@ def clean(self):
             self.cleaned_data["user"] = self.request_user
         if self.errors:
             return
-        if self.cleaned_data['count'] is None:
-            self.cleaned_data["count"] = (
-                self.cleaned_data["user"].get_unit_rows())
+        self.cleaned_data['count'] = min(
+            self.cleaned_data.get("count", 10) or 10,
+            self.cleaned_data["user"].get_unit_rows() or 10)
```

Comment:
in the tests (iirc) they use a non-default value to ensure that its not hardcoding anything.